doc/conflicts.md

```diff
@@ -10,7 +10,7 @@ MVCC data-model is used to do this. This page describes how it works.
 
 ## Replication
 
-The Sync datastore participates in master-less replication with
+The Sync document store participates in master-less replication with
```

Comment:
I've used `DocumentStore` in the docs/comments for my PR (#509) but I'm not sure if it's better than `document store`.  @brynh @ricellis @tomblench thoughts?